includes/core/classes/class-assets.php

```diff
@@ -297,6 +297,7 @@ protected function localize( int $post_id ): array {
 				'currentUser'         => $event->rsvp->get( get_current_user_id() ),
 				'dateTime'            => $event->get_datetime(),
 				'enableAnonymousRsvp' => (bool) get_post_meta( $post_id, 'enable_anonymous_rsvp', true ),
+				'maxAttendance'       => (int) get_post_meta( $post_id, 'max_attendance', true ),
```

Comment:
Let's change this everywhere to `maxAttendanceLimit` and `max_attendance_limit`. Looks good though!